src/features/defi/providers/fox-farming/components/FoxFarmingManager/Withdraw/components/ExpiredWithdraw.tsx

```diff
@@ -50,7 +50,6 @@ export const ExpiredWithdraw: React.FC<StepComponentProps> = ({ onNext }) => {
   const ethAsset = useAppSelector(state => selectAssetById(state, ethAssetId))
   const foxAsset = useAppSelector(state => selectAssetById(state, foxAssetId))
 
-  if (!asset) throw new Error(`Asset not found for AssetId ${opportunity?.underlyingAssetId}`)
   if (!foxAsset) throw new Error(`Asset not found for AssetId ${foxAssetId}`)
   if (!ethAsset) throw new Error(`Asset not found for AssetId ${ethAssetId}`)
```

Comment:
If this is still happening after this PR, we will have to hunt all these and revert them - similar issue in https://github.com/shapeshift/web/issues/3588